README.md

```diff
@@ -1 +1,8 @@
 # EasyContact
+
+### Deploy diagram
+<img src="https://objectstorage.us-ashburn-1.oraclecloud.com/n/idlmil2aymrs/b/static_file/o/rcos%2F%5BEasyContact%5Ddeploy%20diagram.jpg" alt="deploy diagram" />
```

Comment:
Instead of saving the image in S3 bucket and insert the link here, you can try to commit the image in the repository and use image link in Markdown:
`![](your_awesome_img.png)`

Reply:
Good idea! forgot Markdown has built-in image support LOL.